spec/integration/cassandra_record/base_spec.rb

```diff
@@ -103,25 +104,63 @@ def create
     end
   end
 
+  describe ".all" do
+    let(:insert_1) { [9090, 'burgers'] }
+    let(:insert_2) { [8080, 'hot dogs'] }
+    let(:insert_3) { [7070, 'spaghetti'] }
+    let(:insert_4) { [6060, 'tacos'] }
+    let(:insert_5) { [5050, 'birthday cake'] }
+
+    let(:inserts) do
+      [ insert_1,
+        insert_2,
+        insert_3,
+        insert_4,
+        insert_5
+      ]
+    end
+
+    before do
+      inserts.each do |record|
+        insert_statement = <<-CQL
+          INSERT INTO #{keyspace}.test_records (id, name)
+          VALUES (#{record[0]}, '#{record[1]}');
```

Comment:
Yes, but this is test code. I'm not super concerned about optimizing it right now.